packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -477,10 +479,24 @@ export class ConnectionFlow {
     await this.launch()
   }
 
+  /**
+   * Add clientSide attribute to the konnector manifest according to the context
+   *
+   * @todo When the stack handles the clientSide attribute, this won't be needed anymore
+   */
+  withClientSide(konnector) {
+    const clientSideSlugs = get(window, 'cozy.clientSideSlugs', [])
+    const clientSide = clientSideSlugs.includes(konnector.slug)
+
+    return { ...konnector, clientSide }
+  }
```

Comment:
Since the stack handle this attribute can we use it? (https://github.com/cozy/cozy-stack/pull/2976)

Reply:
This attribute is not handled. It is possible to create and run "client" triggers. But if we install a connector with `clientSide` attribute in the manifest, the attribute is still not visible in io.cozy.konnectors doctype.

Reply:
Can you propose what you want to add in the manifest / konnectors doctype? I'm sure that when we have that @nono will be happy to add it when installing the connector.

Reply:
https://github.com/cozy/cozy-stack/issues/3103